<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_sales_report</class>
 <widget class="QWidget" name="admin_sales_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Selections">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>1150</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="salesRepTable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>880</width>
      <height>490</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="dateFilter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>11/12/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/13/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/14/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/15/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/16/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/17/2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11/18/2019</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Filter</string>
    </property>
   </widget>
   <widget class="QComboBox" name="memberFilter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Member Filter</string>
    </property>
   </widget>
   <widget class="QTableView" name="memberTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>240</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkMemberFilter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ENABLE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDateFilter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ENABLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="idAscSort">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Sort ASC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="idDescSort">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Sort DSEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="originalSort">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;FILTERS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;SORTING:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt; SALES REPORT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>159</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to admin page</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+] Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
